extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/discovery/WemoLinkDiscoveryService.java

```diff
@@ -135,7 +142,14 @@ protected void startScan() {
                                 "</DeviceLists>");
 
                         stringParser = StringEscapeUtils.unescapeXml(stringParser);
+                        
+                        // check if there are already paired devices with WeMo Link
+                        if("0".equals(stringParser)) {
+                        	logger.info("There are no devices connected with WeMo Link. Exit discovery");
```

Comment:
Yes, and in order to keep it clean, my suggestion was to have debug log level instead of info.